<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="G-0439" name="orion3_LeadCreation.Lead_InfoCall" tests="6" failures="0" timestamp="18 Jun 2015 09:08:45 GMT" time="127.924" errors="1">
  <testcase name="Leads_ClassificationDetails" time="4.055" classname="orion3_LeadCreation.Lead_InfoCall"/>
  <testcase name="Leads_ProgramOfInterest" time="2.278" classname="orion3_LeadCreation.Lead_InfoCall"/>
  <testcase name="SubmitLead" time="1.760" classname="orion3_LeadCreation.Lead_InfoCall"/>
  <testcase name="Leads_Details" time="3.029" classname="orion3_LeadCreation.Lead_InfoCall"/>
  <testcase name="VerifyLeadInAdmisssionManager" time="109.802" classname="orion3_LeadCreation.Lead_InfoCall">
    <error type="java.lang.AssertionError" message="Lead not found at the top of the new leads table expected [TestNGLNInfoCall_nsscA, TestNGFNInfoCall_nsscA] but found [TestNGLNLiveChat_xWUTvN, TestNGFNLiveChat_xWUTvN]">
      <![CDATA[java.lang.AssertionError: Lead not found at the top of the new leads table expected [TestNGLNInfoCall_nsscA, TestNGFNInfoCall_nsscA] but found [TestNGLNLiveChat_xWUTvN, TestNGFNLiveChat_xWUTvN]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at orion3_LeadCreation.Lead_InfoCall.VerifyLeadInAdmisssionManager(Lead_InfoCall.java:289)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- VerifyLeadInAdmisssionManager -->
  <testcase name="BrowseToAddNewLeadPage" time="7.000" classname="orion3_LeadCreation.Lead_InfoCall"/>
</testsuite> <!-- orion3_LeadCreation.Lead_InfoCall -->
